lib/config/parse-runtime.js

```diff
@@ -42,7 +42,6 @@ module.exports = function(argv, cwd) {
 
             runtimeConfig.useDevServer = true;
             runtimeConfig.devServerHttps = argv.https;
-            runtimeConfig.useHotModuleReplacement = argv.hot || false;
```

Comment:
`hot` now defaults to true. So there is no option at the command line and no need for us to set anything.